mail/api.py

```diff
@@ -154,16 +188,24 @@ def send_individual_email(cls, subject, body, recipient,  # pylint: disable=too-
             recipient (str): email recipient
             sender_address (str): Sender email address
             sender_name (str): Sender name
+            raise_for_status (bool): If true and a non-zero response was received,
 
         Returns:
             requests.Response: response from Mailgun
         """
         # Since .send_batch() returns a list, we need to return the first in the list
         responses = cls.send_batch(subject, body, [recipient], sender_address=sender_address, sender_name=sender_name)
-        return responses[0]
+        _, response, exception = responses[0]
+        if exception is not None:
+            raise exception
```

Comment:
There's no `raise_for_status` for `send_batch`. The `exception` here would come from cases where we were unable to contact Mailgun at all so we won't have a response (or maybe `ImproperlyConfigured` for the 401 case)